web/css/style.scss

```diff
@@ -0,0 +1,272 @@
+$background-color: #1E1E1E;
+$alt-color: #242424;
```

Comment:
I'm not keen on the color changes. The original colorscheme was part of an intentional branding decision and shouldn't be changed by code without discussion. I'll address this in the general review message.

This patch is too large and needs to be broken up into smaller pieces. (If you no longer have the pre-squeezed commits, that's OK -- but if you do, even better.)